lib/options.js

```diff
@@ -37,10 +38,10 @@ module.exports = {
       version: getVersion(),
       external: config.external || config.options.external || {},
       yuidoc: {
-        linkNatives: true,
-        quiet: true,
-        parseOnly: false,
-        lint: false,
+        linkNatives: yuidocOptions.linkNatives || true,
```

Comment:
This prevents `linkNatives: false` from being used (because it would always fall back to `true`).